internal/kojiapi/api/api.go

```diff
@@ -0,0 +1,3 @@
+//go:generate go run github.com/deepmap/oapi-codegen/cmd/oapi-codegen -package=api -generate types,server -o api.gen.go openapi.yml
```

Comment:
Hm `client` was remote from -generate here, do we not use it?

Reply:
If we don't use it, I recommend not generating it. The client code generated by `deepmap` is all over the place. I've opted doing it manually in the `worker` package as well.

Reply:
The code that used the client api was dropped (it was an unused test). Generating it still was a left-over.